test/clients/reports_test.exs

```diff
@@ -91,4 +91,87 @@ defmodule GlassFactoryApi.Clients.ReportsTest do
       end
     end
   end
+
+  describe "list_rates_and_costs_reports/3" do
+    test "returns reports of the given client id", %{bypass: bypass, config: config} do
+      request_response = GlassFactoryApi.Fixtures.Clients.Reports.list_rates_and_costs_reports()
+
+      Bypass.expect_once(bypass, fn conn ->
+        conn
+        |> Plug.Conn.put_resp_content_type("application/json")
+        |> Plug.Conn.resp(200, request_response)
+      end)
+
+      assert {
```

Comment:
🎨 we can break this down into smaller assertions to ease debugging and avoid too much indentation:

```elixir
expected = %RatesAndCostsReport{
  client_id: 1234,
  project_id: 12345,
  job_id: nil,
  activity_id: 123_456,
  user_id: 222,
  role_id: 1234,
  date: "2018-06-19",
  time: 8,
  rate: 35,
  cost: 35
}

assert {:ok, [report]} == Reports.list_rates_and_costs_reports(2079, [], config)

assert expected == report
```

Reply:
:til:

Reply:
Omg, this is beautiful ✨, Thanks @britto 😄